uktrade/enquiry-mgmt-tool
# Truncate enquiry text in list view

## Description of change

Currently when viewing a list of enquiries, the whole enquiry text is showing. This can often be extremely long and so takes up a lot of the page.

This PR truncates the enquiry text when it is longer than 250 characters. 

I have used a custom template filter for this as Django's built-in `truncatechars` will cut off a string mid-word.

I have adjusted two existing enquiry fixtures in order to test this functionality.

## Test instructions

- Search for this company - ABC Electronics Co. (long text)
- While still in the list view, check that the enquiry text is cut off after 250 characters but that the final word intact
